dev/NavigationView/NavigationView.cpp

```diff
@@ -1115,6 +1119,51 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
+{
+    if (PaneDisplayMode() == winrt::NavigationViewPaneDisplayMode::Top
+        || DisplayMode() == winrt::NavigationViewDisplayMode::Minimal)
+    {
+        if (auto backButton = m_backButton.try_as<winrt::Button>())
```

Comment:
locals should be const when possible [https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#con4-use-const-to-define-objects-with-values-that-do-not-change-after-construction](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#con4-use-const-to-define-objects-with-values-that-do-not-change-after-construction)

Reply:
What exactly should be constant here? The "newly" assigned backButton? 😅

Reply:
That's right, the local varabile named backButton is a pointer which does not change and thus should be const.

---
In reply to: [387336884](https://github.com/microsoft/microsoft-ui-xaml/pull/2051#discussion_r387336884) [](ancestors = 387336884)